app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        today = datetime.date.today()
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+
+        if visitor["vaccine"]["expiration_date"] < today:
+            raise OutdatedVaccineError("Vaccine is expired.")
+
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError("Visitor is not wearing a mask.")
+
+        return f"Welcome to {self.name}"
```

Comment:
The `visit_cafe` method is a bit long and does several things. You might consider breaking it down into smaller methods each doing one thing. This will make your code more readable and maintainable. Here's a suggestion:

```python
def check_vaccination_status(self, visitor: dict) -> None:
    if "vaccine" not in visitor:
        raise NotVaccinatedError("Visitor is not vaccinated.")
    if visitor["vaccine"]["expiration_date"] < datetime.date.today():
        raise OutdatedVaccineError("Vaccine is expired.")

def check_mask(self, visitor: dict) -> None:
    if not visitor.get("wearing_a_mask", False):
        raise NotWearingMaskError("Visitor is not wearing a mask.")

def visit_cafe(self, visitor: dict) -> str:
    self.check_vaccination_status(visitor)
    self.check_mask(visitor)
    return f"Welcome to {self.name}"
```
This is just a suggestion, and your current code works fine as well. But remember, clean code is not just about making code work, but also about making it easy for others (and future you) to read and understand.